features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/root/PreferencesRootNode.kt

```diff
@@ -73,7 +84,16 @@ class PreferencesRootNode @AssistedInject constructor(
             onOpenAnalytics = this::onOpenAnalytics,
             onOpenAbout = this::onOpenAbout,
             onVerifyClicked = this::onVerifyClicked,
-            onOpenDeveloperSettings = this::onOpenDeveloperSettings
+            onOpenDeveloperSettings = this::onOpenDeveloperSettings,
+            onSuccessLogout = { onSuccessLogout(activity, it) },
+            onManageAccountClicked = { onManageAccountClicked(activity, state.accountManagementUrl) },
         )
     }
+
+    private fun onSuccessLogout(activity: Activity, url: String?) {
+        Timber.d("Success logout with result url: $url")
+        url?.let {
+            activity.openUrlInChromeCustomTab(null, false, it)
```

Comment:
What is this new URL used for? Could we add some docs? 

Also, on lambdas you can add names to the parameters: they don't really do anything AFAIK, but are useful for self--documented code. Maybe we could use something like `onSuccessLogout: (fooUrl: String?) -> Unit`?